app/models/map.rb

```diff
@@ -58,6 +58,7 @@ class Phase < Enum
         mixed:      {name: "Mixed"},
         skywars:    {name: "Skywars"},
         survival:   {name: "Survival Games"},
+        survival:   {name: "Payload"},
```

Comment:
Shouldn't this be `payload: {name: "Payload"},`

Reply:
Whoops, yea will fix